test/cmdlineTests/standard_ewasm_requested/output.json.orig

```diff
@@ -0,0 +1,171 @@
+<<<<<<< HEAD
```

Comment:
a leftover file?

Reply:
hmm.. maybe we should add a check in style check that ensures that there are no ".orig" files

Reply:
or `.gitignore`